monero-harness/src/image.rs

```diff
@@ -229,6 +229,7 @@ impl IntoIterator for MoneroWalletRpcArgs {
             format!("--daemon-address={}", self.daemon_address),
             format!("--rpc-bind-port={}", RPC_PORT),
             "--log-level=4".to_string(),
+            "--allow-mismatched-daemon-version".to_string(), /* https://github.com/monero-project/monero/issues/8600 */
```

Comment:
Did you experience the problem described in the issue yourself?

Reply:
Yeah exactly